packages/block-library/src/buttons/test/edit.native.js

```diff
@@ -66,14 +66,20 @@ describe( 'when a button is shown', () => {
 		);
 		fireEvent.press( settingsButton );
 
-		const radiusSlider = await waitFor( () =>
-			getByTestId( 'Slider Border Radius' )
+		const radiusStepper = await waitFor( () =>
+			getByA11yLabel( /Border Radius/ )
 		);
-		fireEvent( radiusSlider, 'valueChange', '25' );
+
+		const incrementButton = await waitFor( () =>
+			within( radiusStepper ).getByTestId( 'Increment' )
+		);
+		fireEvent.press( incrementButton );
```

Comment:
I noticed that the `Stepper` component uses the `pressIn` and `pressOut` events instead of the regular `press` event:
https://github.com/WordPress/gutenberg/blob/ee280e131c35037a05d07adfeac4234475736233/packages/components/src/mobile/bottom-sheet/stepper-cell/stepper.android.js#L61-L62

So we would need to update this call to:
```
fireEvent( incrementButton, 'onPressIn' );
```